ui/tests/integration/modifiers/code-mirror-test.ts

```diff
@@ -0,0 +1,65 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { fillIn, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Modifier | code-mirror', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    let valueToChange = 'initial value in editor';
+    this.set('value', valueToChange);
+    this.set('onInput', () => {});
```

Comment:
I don’t get why, but ESLint is complaining about these void functions. I say we just disable that rule of this file by adding this at the top of the file:

```ts
/* eslint-disable @typescript-eslint/no-empty-function */
```